src/Microsoft.ApplicationInsights/DataContracts/StackFrame.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Microsoft.ApplicationInsights.DataContracts
+{
+    using System;
+
+    /// <summary>
+    /// Wrapper class for <see cref="Extensibility.Implementation.External.StackFrame"/> for API exposure.
+    /// </summary>
+    public sealed class StackFrame
```

Comment:
>StackFrame [](start = 24, length = 10)

is this class needed instead of using `System.Diagnostics.StackFrame`?

Reply:
Yeah our StackFrame has some stuff that BCL's version doesn't, like "level" or "method"...